pkg/operator/controllers/genevalogging/const.go

```diff
@@ -73,7 +73,7 @@ const (
 [FILTER]
 	Name grep
 	Match containers
-	Regex NAMESPACE ^(?:default|kube-.*|openshift|(?!openshift-(logging|gitops|user-workload-monitoring))(openshift-.*))$
+	Regex NAMESPACE ^(?:default|kube-.*|openshift|(?!openshift-(logging|gitops|user-workload-monitoring|adp|distributed-tracing|cluster-samples-operator|cnv|serverless|pipelines|nfd))(openshift-.*))$
```

Comment:
The samples operator is a part of the platform

Reply:
Fair enough. I haven't seen a time we've needed to use the logs of it while on-call, but that's probably a different discussion (trimming cx things vs trimming things that aren't useful)